panic due to conflicting flags

I downloaded the `operations-center` CLI from the latest main build here, but not sure when this issue was introduced:
- https://github.com/FuturFusion/operations-center/actions/runs/19562664659

Currently getting this when I try to use the `admin` sub-command:
```
~ operations-center admin os system backup
panic: unable to redefine 'd' shorthand in "backup" flagset: it's already used for "data" flag

goroutine 1 [running]:
github.com/spf13/pflag.(*FlagSet).AddFlag(0xc00021f300, 0xc0001fc1e0)
	/home/runner/go/pkg/mod/github.com/spf13/pflag@v1.0.10/flag.go:904 +0x3f4
github.com/spf13/cobra.(*Command).mergePersistentFlags.(*FlagSet).AddFlagSet.func2(0xc0001fc1e0)
	/home/runner/go/pkg/mod/github.com/spf13/pflag@v1.0.10/flag.go:917 +0x3f
github.com/spf13/pflag.(*FlagSet).VisitAll(0xc000234608?, 0xc0001c9aa0)
	/home/runner/go/pkg/mod/github.com/spf13/pflag@v1.0.10/flag.go:320 +0xd5
github.com/spf13/pflag.(*FlagSet).AddFlagSet(...)
	/home/runner/go/pkg/mod/github.com/spf13/pflag@v1.0.10/flag.go:915
github.com/spf13/cobra.(*Command).mergePersistentFlags(0xc000234608)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.1/command.go:1901 +0x94
github.com/spf13/cobra.(*Command).InitDefaultHelpFlag(0xc000234608)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.1/command.go:1220 +0x1f
github.com/spf13/cobra.(*Command).execute(0xc000234608, {0x13886a0, 0x0, 0x0})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.1/command.go:916 +0x105
github.com/spf13/cobra.(*Command).ExecuteC(0xc000200008)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.1/command.go:1148 +0x46f
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.1/command.go:1071
main.main0({0xc000020060, 0x4, 0x4}, {0xdd7ce0, 0xc00006a038}, {0xdd7ce0, 0xc00006a040}, {0x7fd7294a14b8, 0xc00007ff20})
	/home/runner/work/operations-center/operations-center/cmd/operations-center/main.go:110 +0x6e5
main.main()
	/home/runner/work/operations-center/operations-center/cmd/operations-center/main.go:24 +0x12f


~ operations-center admin os application backup
panic: unable to redefine 'd' shorthand in "backup" flagset: it's already used for "data" flag

goroutine 1 [running]:
github.com/spf13/pflag.(*FlagSet).AddFlag(0xc00023ce00, 0xc00021a320)
	/home/runner/go/pkg/mod/github.com/spf13/pflag@v1.0.10/flag.go:904 +0x3f4
github.com/spf13/cobra.(*Command).mergePersistentFlags.(*FlagSet).AddFlagSet.func2(0xc00021a320)
	/home/runner/go/pkg/mod/github.com/spf13/pflag@v1.0.10/flag.go:917 +0x3f
github.com/spf13/pflag.(*FlagSet).VisitAll(0xc00024f508?, 0xc0001e1aa0)
	/home/runner/go/pkg/mod/github.com/spf13/pflag@v1.0.10/flag.go:320 +0xd5
github.com/spf13/pflag.(*FlagSet).AddFlagSet(...)
	/home/runner/go/pkg/mod/github.com/spf13/pflag@v1.0.10/flag.go:915
github.com/spf13/cobra.(*Command).mergePersistentFlags(0xc00024f508)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.1/command.go:1901 +0x94
github.com/spf13/cobra.(*Command).InitDefaultHelpFlag(0xc00024f508)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.1/command.go:1220 +0x1f
github.com/spf13/cobra.(*Command).execute(0xc00024f508, {0x13886a0, 0x0, 0x0})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.1/command.go:916 +0x105
github.com/spf13/cobra.(*Command).ExecuteC(0xc000220008)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.1/command.go:1148 +0x46f
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.10.1/command.go:1071
main.main0({0xc0000c8010, 0x4, 0x4}, {0xdd7ce0, 0xc0000b2030}, {0xdd7ce0, 0xc0000b2038}, {0x7fea15e5e3b8, 0xc000111e80})
	/home/runner/work/operations-center/operations-center/cmd/operations-center/main.go:110 +0x6e5
main.main()
	/home/runner/work/operations-center/operations-center/cmd/operations-center/main.go:24 +0x12f
```

Its impacting multiple sub commands of `admin` but some are working:
```
~ operations-center admin os application list  
WARNING: The IncusOS API and configuration is subject to change

+-------------------+
|       NAME        |
+-------------------+
| operations-center |
+-------------------+
```

Looks like it might be something to do with imported commands from `incus-osd` built with `cmdGenericRun` as they have a `-d/--data` defined, maybe its conflicting with the `operations-center` global `-d/--debug` flag.

--- a/cmd/operations-center/main.go
+++ b/cmd/operations-center/main.go
@@ -69,7 +69,7 @@ func main0(args []string, stdout io.Writer, stderr io.Writer, env env) error {
 	app.PersistentPreRunE = globalCmd.PreRun
 	app.PersistentFlags().BoolVar(&globalCmd.flagVersion, "version", false, "Print version number")
 	app.PersistentFlags().BoolVarP(&globalCmd.flagHelp, "help", "h", false, "Print help")
-	app.PersistentFlags().BoolVarP(&globalCmd.flagLogDebug, "debug", "d", false, "Show all debug messages")
+	app.PersistentFlags().BoolVar(&globalCmd.flagLogDebug, "debug", false, "Show all debug messages")
 	app.PersistentFlags().BoolVarP(&globalCmd.flagLogVerbose, "verbose", "v", false, "Show all information messages")
 	app.PersistentFlags().BoolVar(&globalCmd.flagForceLocal, "force-local", false, "Force using the local unix socket")
 

